src/ethereum_clis/clis/nimbus.py

```diff
@@ -66,69 +67,62 @@ class NimbusExceptionMapper(ExceptionMapper):
     """Translate between EEST exceptions and error strings returned by Nimbus."""
 
     mapping_substring: ClassVar[Dict[ExceptionBase, str]] = {
-        TransactionException.TYPE_4_TX_CONTRACT_CREATION: (
-            "set code transaction must not be a create transaction"
+        # TODO: these need to be fixed / incorrectly mapped
+        TransactionException.INSUFFICIENT_MAX_FEE_PER_GAS: (
+            "zero gasUsed but transactions present"
         ),
-        TransactionException.INSUFFICIENT_ACCOUNT_FUNDS: "invalid tx: not enough cash to send",
         TransactionException.TYPE_3_TX_MAX_BLOB_GAS_ALLOWANCE_EXCEEDED: (
-            "would exceed maximum allowance"
+            "zero gasUsed but transactions present"
+        ),
+        TransactionException.TYPE_4_TX_CONTRACT_CREATION: (
+            "zero gasUsed but transactions present"
+        ),
+        TransactionException.TYPE_4_EMPTY_AUTHORIZATION_LIST: (
+            "zero gasUsed but transactions present"
+        ),
+        TransactionException.TYPE_4_TX_PRE_FORK: (
+            "zero gasUsed but transactions present"
         ),
         TransactionException.INSUFFICIENT_MAX_FEE_PER_BLOB_GAS: (
-            "max fee per blob gas less than block blob gas fee"
+            "zero gasUsed but transactions present"
```

Comment:
there must be a validation for the map that does not allow to have 2 different exception codes with the same exception string/pattern. otherwise we can invent redundant exception codes and forget.